docs/content/preview/explore/fault-tolerance/transaction-availability.md

```diff
@@ -0,0 +1,234 @@
+a9b4675fdaaa4d4b949adc5e53d183bf   [0x5555, 0xAAA9]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a8c50129f63642459a02ed4ee492a1f3   [0x0000, 0x5554]  127.0.0.1:9100  6789e52b1c334844a66078fe9fdf95fa
+```   
+Commit the transaction
+```output.sql
+yugabyte@yugabyte=# UPDATE txndemo set k2=30 where k1=1; COMMIT;
+UPDATE 1
+Time: 1728.246 ms (00:01.728)
+COMMIT
+Time: 2.964 ms
+```    
+You can see that even though the node `127.0.0.2` failed before receiving the provisional write, the transaction has succeeded. This is because a new leader (node:`127.0.0.3`) was quickly elected.
+    
+## Failure of Transaction Manager(Coordinator)
+The node to which a client connects to, acts as the coordinator for the transaction. You have seen how YugabyteDB is inherently resilient to node failures in the above two scenarios. In this example you will see how a transaction will abort when the coordinator fails. More details on the role of the transaction manager can be found in [Transactional I/O](../../architecture/transactions/transactional-io-path/#client-requests-transaction) section.
+
+List the nodes via `yb-admin list_tablets ysql.yugabyte txndemo`
+```
+Tablet-UUID                       Key Range         Leader-IP       Leader-UUID
+7e2dfb66a4654aa5b2fb133b446aaabc  [0x0000, 0x5554]  127.0.0.2:9100  4739b43f76184e1cab003b88686df290
+a9b4675fdaaa4d4b949adc5e53d183bf  [0x5555, 0xAAA9]  127.0.0.3:9100  7402fbc9c6384d80bb7bcd09b89dbca9
+a8c50129f63642459a02ed4ee492a1f3  [0x0000, 0x5554]  127.0.0.1:9100  6789e52b1c334844a66078fe9fdf95fa
+```    
+Connect to `127.0.0.3` and start a simple transaction to update the value of row `k1=1` to `40`.
+```output.sql
+ysqlsh -h 127.0.0.3  -U yugabyte -d yugabyte
+yugabyte@yugabyte=# BEGIN;
+BEGIN
+Time: 2.047 ms
+yugabyte@yugabyte=# UPDATE txndemo set k2=40 where k1=1;
+UPDATE 1
+Time: 50.624 ms
+ ```
+
+Stop the node:`127.0.0.3`. _Note_: We are stopping `127.0.0.3` as that is the node that we have connected to.
+```sh
+yugabyted stop --base_dir=/tmp/ybd3
+```
+
+Commit the transaction
+```output.sql
+yugabyte@yugabyte=# COMMIT;
+FATAL:  57P01: terminating connection due to unexpected postmaster exit
+server closed the connection unexpectedly
+  This probably means the server terminated abnormally
+  before or while processing the request.
+The connection to the server was lost. Attempting reset: Failed.
+Time: 2.499 ms
+```
+
+The client received an error response from the server. Connect to a different node and check the value.
+```sql
+ysqlsh -h 127.0.0.1  -U yugabyte -d yugabyte
+select * from txndemo where k1=1;
+```
+```output.sql
+ k1 | k2
+----+----
+  1 | 30
+(1 row)
+```
+
+The row did not get the intended value of `40` and still has the old value of `30`. The transaction did not go through. When the transaction coordinator fails before commit happens, the transaction is lost. At this point, it will be the application's responsibility to retry the transaction.
+
```

Comment:
is it worth showing an example where the client node fails and the rows it had provisional records on quickly become free to update in another session?

Reply:
It is certainly within the context, but techincally that would be client failure. This page is about server side failures. Let us ask the docs folks .. @aishwarya24 , @polarweasel , @prasad-rad - any suggestions here ?

Reply:
Do you want to split this into two pages, one for node failures and one for client failures? That might be the best way to keep this page on-topic while adding some useful client-side examples.

Reply:
I'll be addressing the client failures and retries in a separate page ..